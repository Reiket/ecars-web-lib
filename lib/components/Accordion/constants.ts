export const ACCORDION_TEST_ID = 'accordionTestId';
