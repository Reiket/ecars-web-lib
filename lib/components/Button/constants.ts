import type {ButtonHTMLAttributes} from 'react';

export const BUTTON_SIZE = Object.freeze({
  BIG: 'big',
  SMALL: 'small',
});

export const BUTTON_COLOR = Object.freeze({
  GREEN: 'green',
  WHITE: 'white',
  GRAY: 'gray',
});


export type ButtonAttributesType = ButtonHTMLAttributes<HTMLButtonElement>['type']
export type ButtonSizeType = (typeof BUTTON_SIZE)[keyof typeof BUTTON_SIZE];
export type ButtonColorType = (typeof BUTTON_COLOR)[keyof typeof BUTTON_COLOR];

export const BUTTON_TEST_ID = 'buttonTestId';