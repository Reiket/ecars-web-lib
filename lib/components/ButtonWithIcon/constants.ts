export const LEFT_ICON_BUTTON_TEST_ID = 'leftIconButtonTestId';
export const RIGHT_ICON_BUTTON_TEST_ID = 'rightIconButtonTestId';
