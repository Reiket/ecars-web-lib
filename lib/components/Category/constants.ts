export const CATEGORIES_LIST = Object.freeze({
  GUIDES: 'guides',
  REVIEWS: 'reviews',
  NEWS: 'news',
});

export type CategoriesListType = (typeof CATEGORIES_LIST)[keyof typeof CATEGORIES_LIST];

export const CATEGORY_TEST_ID = 'categoryTestId';
