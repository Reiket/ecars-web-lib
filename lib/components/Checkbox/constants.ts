import type {ElementProps} from '@/services/types';

export interface CheckboxProps extends ElementProps {
  name?: string;
  id?: string;
  disabled?: boolean;
  onChange?: (checked: boolean) => void;
  checked?: boolean;
  hasError?: boolean;
}

export const CHECKBOX_TEST_ID = 'checkboxTestId';
export const CHECKBOX_TEST_INPUT_ID = 'checkboxTestInputId';
