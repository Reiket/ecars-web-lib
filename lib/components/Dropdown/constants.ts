import type {ReactNode} from 'react';

export interface DropdownOption {
  value: string;
  label?: ReactNode;
}

export interface DropdownProps {
  onSelect: (value: string) => void;
  category: string;
  options: DropdownOption[];
}

export const DROPDOWN_TEST_ID = 'dropdownTestId';

export const DROPDOWN_THEME = Object.freeze({
  DARK: 'dark',
  LIGHT: 'light',
});

export type DropdownThemeType = (typeof DROPDOWN_THEME)[keyof typeof DROPDOWN_THEME];
