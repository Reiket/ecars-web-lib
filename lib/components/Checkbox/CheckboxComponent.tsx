import type {FC} from 'react';
import type {CheckboxProps} from '@/components/Checkbox/constants';
import {Checkbox} from '@/components/Checkbox/constants';

export const CheckboxComponent: FC<CheckboxProps> = ({id, block, ...props}) => (
  <Checkbox.Block
    htmlFor={id}
    block={block}
  >
    <Checkbox.Input
      {...props}
      id={id}
    />
    <Checkbox.Indicator />
  </Checkbox.Block>
);
