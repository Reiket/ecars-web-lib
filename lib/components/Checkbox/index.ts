import {CheckboxComponent} from '@/components/Checkbox/CheckboxComponent';
import {CheckboxBlock} from '@/components/Checkbox/components/CheckboxBlock';
import {CheckboxIndicator} from '@/components/Checkbox/components/CheckboxIndicator';
import {CheckboxInput} from '@/components/Checkbox/components/CheckboxInput';

export const Checkbox = Object.assign(CheckboxComponent, {
  Block: CheckboxBlock,
  Indicator: CheckboxIndicator,
  Input: CheckboxInput,
});
