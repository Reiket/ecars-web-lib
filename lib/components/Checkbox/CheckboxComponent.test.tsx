import {fireEvent, render, screen} from '@testing-library/react';
import React from 'react';
import {Checkbox, CHECKBOX_TEST_ID, CHECKBOX_TEST_INPUT_ID} from '@/components/Checkbox/constants';

describe('Checkbox component', () => {
  const disabledOptions = [true, false];
  const errorOptions = [true, false];
  const checkedOptions = [true, false];
  disabledOptions.forEach((disabled) => {
    checkedOptions.forEach((checked) => {
      errorOptions.forEach((error) => {
        const disabledStatus = disabled ? 'is disabled' : 'is not disabled';
        const checkedStatus = checked ? 'is checked' : 'is not checked';
        const errorStatus = error ? 'has an error and input has an error class' : 'has no error';
        const testName = `renders checkbox ${disabledStatus}, ${checkedStatus}, ${errorStatus}`;
        test(testName, () => {
          const props = {
            checked: checked,
            disabled: disabled,
            hasError: error,
          };
          const {container} = render(<Checkbox {...props} />);
          const checkboxElement = screen.getByTestId(CHECKBOX_TEST_ID);
          const inputElement = screen.getByTestId(CHECKBOX_TEST_INPUT_ID) as HTMLInputElement;

          if (error) {
            expect(inputElement).toHaveClass('error');
          }
          expect(inputElement.checked).toBe(checked);
          expect(inputElement.disabled).toBe(disabled);
          expect(checkboxElement).toBeInTheDocument();
          expect(container).toMatchSnapshot();
        });
      });
    });
  });
  test('onChange event changed checked when user clicks on checkbox', () => {
    const props = {
      checked: false,
      onChange: jest.fn(),
    };
    render(<Checkbox {...props} />);
    const inputElement = screen.getByTestId(CHECKBOX_TEST_INPUT_ID) as HTMLInputElement;
    expect(inputElement.checked).toBe(false);
    fireEvent.change(inputElement, {target: {checked: true}});
    expect(inputElement.checked).toBe(true);
  });
});
