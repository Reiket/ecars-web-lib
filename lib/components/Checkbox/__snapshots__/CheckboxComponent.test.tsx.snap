// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Checkbox component > renders checkbox is disabled, is checked, has an error and input has an error class 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      checked=""
      class="checkbox__input error"
      data-testid="checkboxTestInputId"
      disabled=""
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component > renders checkbox is disabled, is checked, has no error 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      checked=""
      class="checkbox__input"
      data-testid="checkboxTestInputId"
      disabled=""
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component > renders checkbox is disabled, is not checked, has an error and input has an error class 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      class="checkbox__input error"
      data-testid="checkboxTestInputId"
      disabled=""
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component > renders checkbox is disabled, is not checked, has no error 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      class="checkbox__input"
      data-testid="checkboxTestInputId"
      disabled=""
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component > renders checkbox is not disabled, is checked, has an error and input has an error class 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      checked=""
      class="checkbox__input error"
      data-testid="checkboxTestInputId"
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component > renders checkbox is not disabled, is checked, has no error 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      checked=""
      class="checkbox__input"
      data-testid="checkboxTestInputId"
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component > renders checkbox is not disabled, is not checked, has an error and input has an error class 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      class="checkbox__input error"
      data-testid="checkboxTestInputId"
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component > renders checkbox is not disabled, is not checked, has no error 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      class="checkbox__input"
      data-testid="checkboxTestInputId"
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component renders checkbox is disabled, is checked, has an error and input has an error class 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      checked=""
      class="checkbox__input error"
      data-testid="checkboxTestInputId"
      disabled=""
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component renders checkbox is disabled, is checked, has no error 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      checked=""
      class="checkbox__input"
      data-testid="checkboxTestInputId"
      disabled=""
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component renders checkbox is disabled, is not checked, has an error and input has an error class 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      class="checkbox__input error"
      data-testid="checkboxTestInputId"
      disabled=""
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component renders checkbox is disabled, is not checked, has no error 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      class="checkbox__input"
      data-testid="checkboxTestInputId"
      disabled=""
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component renders checkbox is not disabled, is checked, has an error and input has an error class 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      checked=""
      class="checkbox__input error"
      data-testid="checkboxTestInputId"
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component renders checkbox is not disabled, is checked, has no error 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      checked=""
      class="checkbox__input"
      data-testid="checkboxTestInputId"
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component renders checkbox is not disabled, is not checked, has an error and input has an error class 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      class="checkbox__input error"
      data-testid="checkboxTestInputId"
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;

exports[`Checkbox component renders checkbox is not disabled, is not checked, has no error 1`] = `
<div>
  <label
    class="checkbox"
    data-testid="checkboxTestId"
    for="checkbox"
  >
    <input
      class="checkbox__input"
      data-testid="checkboxTestInputId"
      id="checkbox"
      name="checkbox"
      type="checkbox"
    />
    <span
      class="checkbox__indicator"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        stroke-width="0"
        viewBox="0 0 448 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M438.6 105.4c12.5 12.5 12.5 32.8 0 45.3l-256 256c-12.5 12.5-32.8 12.5-45.3 0l-128-128c-12.5-12.5-12.5-32.8 0-45.3s32.8-12.5 45.3 0L160 338.7 393.4 105.4c12.5-12.5 32.8-12.5 45.3 0z"
        />
      </svg>
    </span>
  </label>
</div>
`;
